portability-core/src/main/java/org/dataportabilityproject/shared/ImageStreamProvider.java

```diff
@@ -0,0 +1,16 @@
+package org.dataportabilityproject.shared;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.net.HttpURLConnection;
+import java.net.URL;
+
+public class ImageStreamProvider {
```

Comment:
nit: can you add documentation for the class, as well as the license header?

Reply:
Adding light documentation.  I imagine more documentation will be added as the class grows.

Reply:
can you make it a bit more descriptive? like what the stream is used for etc.

Reply:
Sorry to nitpick but I think it is still unclear to a reader what an image stream is exactly.. How about something like this:

Gets an input stream to an image, given its URL. Used by {@link FlickrPhotoService} to upload the image.